MH-FiorellaTheme5/config.ru

```diff
@@ -0,0 +1,30 @@
+require "rack"
```

Comment:
Why you have this file in this Pull Request?

Reply:
This files are necessary to deploy to heroku, and they were in my local repository and i decided to added them